ROADMAP.md

```diff
@@ -1,16 +1,37 @@
-# KServe 2023 Roadmap
+# KServe 2024-2025 Roadmap
+## Objective: "Support GenAI inference"
+- LLM Serving Runtimes
+   * Support Speculative Decoding with vLLM runtime.
+   * Support LoRA adapters.
+   * Support LLM Serving runtimes for TensorRT-LLM, TGI and provide benchmarking comparisions.
+   * Support multi-host, multi-GPU inference runtime.
+
+- LLM Autoscaling
+   * Support Model Caching with automatic PV/PVC provisioning.
+   * Support Autoscaling settings for serving runtimes.
+   * Support Autoscaling based on custom metrics.
+
+- LLM RAG Pipeline Orchestration
+   * Support declarative RAG with KServe Inference Graph.
+
+- GenAI Task APIs
+   * Community maintained Open Inference Protocol repo for OpenAI schema.
+   * Support verical GenAI Task APIs such as embedding, Text-to-Image, Text-To-Code, Doc-To-Text.
+
+- LLM Gateway
+   * Support multiple LLM providers.
+   * Support token based rating limiting.
+   * Support LLM router with traffic shaping, fallback, load balancing.
 
 ## Objective: "Graduate core inference capability to stable/GA"
-- Promote `InferenceService` and `ClusterServingRuntime`/`ServingRuntime` CRD from v1beta1 to v1 
+- Promote `InferenceService` and `ClusterServingRuntime`/`ServingRuntime` CRD to v1
   * Improve `InferenceService` CRD for REST/gRPC protocol interface
   * Unify model storage spec and implementation between KServe and ModelMesh
-  * Add Status to `ServingRuntime` for both ModelMesh and KServe, surface `ServingRuntime` validation errors and deployment status
-  * Deprecate `TrainedModel` CRD and use `InferenceService` annotation to allow dynamic model updates as alternative option to storage initializer
-  * Collocate transformer and predictor in the pod to reduce sidecar resources and networking latency
+  * Deprecate `TrainedModel` CRD and `InferenceService` to support multiple models for model collocations, LoRA adapters.
```

Comment:
Does this mean that `InferenceService` is planned to become deprecated ? I don't think so, but the wording here is misleading.

Reply:
I think the intention here is to _**extend**_  `InferenceService` to support other types of mechanisms and hence it's an item under this "Promote `InferenceService`...CRD to v1" section

@yuzisun to double check